### DISCUSSION TITLE
Theming - Custom dark background

### DISCUSSION THREAD

**Discussion by Jopp-gh** [linuxmint/discussions#624] [Questions]
Basically, I'm a bit tired to see extreme  black themes (Mint cinnamon). And custom themes change too much for me, so I'd like to ask the Mint community for help to tweak the `application` tab and the `desktop` tab under "settings>themes". 

What I'm after are all variables that define a dark background. CSS isn't an issue, with the right tools, like an election console.
With Mint, there's no such interactive console so I've zero clue how to theme Mint.

Also, there are discrepancies  even in Mint-y themes, only the orange and the green theme support dark backgrounds in all kind of applications - even those apps who don't have a theme setting. 

To start with, I could take a look at Mint-Y, that wouldn't be the first time, as these records are massive and demoralizing to study. 

I need just a few variables, to tone up the green and dark version of Mint-Y. Help is greatly appreciated!

**Comment by Secret-chest** [comment]
It won't look the same but you can use Themix to generate a theme with your own colours.

**Reply by Jopp-gh** [reply]
Awesome !
 
I installed the flatpak from MintInstall and my terminal complains some missing dependencies :
Error: `runtime/org.gnome.Sdk/x86_64/44`

Unlucky, I've no clue where to get and how to install this dependency, so Themix is able to use it. Often, old apps have  this behavior and even if you fix one error, there are more to follow.  

I checked their GitHub site in the hope to find a deb package and saw the ppa package is outdated and no more maintained. Not very inspiring...

**Reply by Secret-chest** [reply]
> Awesome !
> 
> I installed the flatpak from MintInstall and my terminal complains some missing dependencies : Error: `runtime/org.gnome.Sdk/x86_64/44`
> 
> Unlucky, I've no clue where to get and how to install this dependency, so Themix is able to use it. Often, old apps have this behavior and even if you fix one error, there are more to follow.
> 
> I checked their GitHub site in the hope to find a deb package and saw the ppa package is outdated and no more maintained. Not very inspiring...

Check this repository: <https://github.com/themix-project/themix-gui>

**Comment by Jopp-gh** [comment]
Well, that's the repo I was checking last time. The packages are outdated. 

I'm sure the folks who made custom themes didn't code their CSS from the top of their mind. 

So, what tools did they use?
